cmd/vic-machine/configure/configure.go

```diff
@@ -514,6 +515,15 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 		}
 	}
 
+	if c.ContainerCount != nil {
```

Comment:
Do we have a null pointer scenario? If so, should we set the ContainerCount to 0 for this case?

Reply:
When we configure VCH, but do not touch container count, or any other options, these options will be nil. So not null indicates users have reset this option from command line.